README.md

```diff
@@ -1,8 +1,8 @@
 # Kubeflow Trainer
 
-[![Build Status](https://github.com/kubeflow/training-operator/actions/workflows/test-go.yaml/badge.svg?branch=master)](https://github.com/kubeflow/training-operator/actions/workflows/test-go.yaml?branch=master)
-[![Coverage Status](https://coveralls.io/repos/github/kubeflow/training-operator/badge.svg?branch=master)](https://coveralls.io/github/kubeflow/training-operator?branch=master)
-[![Go Report Card](https://goreportcard.com/badge/github.com/kubeflow/training-operator)](https://goreportcard.com/report/github.com/kubeflow/training-operator)
+[![Build Status](https://github.com/kubeflow/trainer/actions/workflows/test-go.yaml/badge.svg?branch=master)](https://github.com/kubeflow/trainer/actions/workflows/test-go.yaml?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/kubeflow/trainer/badge.svg?branch=master)](https://coveralls.io/github/kubeflow/trainer?branch=master)
```

Comment:
As I can see the badge doesn't show actual information: https://github.com/kubeflow/trainer/blob/master/README.md?plain=1#L4

Should we remove the branch ?
```
https://coveralls.io/repos/github/kubeflow/trainer/badge.svg
```

Reply:
cc @azzamh15 @IRONICBo

Reply:
Now the `master` CI is not finished, I think we need wait for the CI reporting to coverage.

<img width="503" alt="image" src="https://github.com/user-attachments/assets/a88310d4-4705-47cc-994a-13bed5abba04" />


cc @andreyvelich

Reply:
Oh, it works now! Thank you for your help!